crossplane/crossplane
# Support setting Pod Annotations in ControllerConfig

### Description of your changes



Adds support for setting annotations on a provider Pod via the
referenced ControllerConfig. This is especially useful in cases where a
mutating webhook acts on Pods based on annotations present.

Signed-off-by: hasheddan <georgedanielmangum@gmail.com>

I have:


### How has this code been tested



1. Install Crossplane
2. Create `Provider` that references `ControllerConfig` with `metadata.annotations` set
3. Observe annotations present on `Pod`

[contribution process]: https://git.io/fj2m9
